src/services/messages-service.js

```diff
@@ -22,6 +24,21 @@ export const sendMessage = async (sender, recipient, message, attachment) => {
   return response.data;
 }
 
+
+/**
+* delete a message
+* @param message message to send
+*/
+export const deleteMessage = async (mid) => {
+  console.log(mid)
```

Comment:
Please remove console.log and commented code. Also in JSDoc it should be `@param mid`